Makefile

```diff
@@ -125,6 +125,10 @@ doc: $(MANIFEST)
 	antsibull-docs collection --use-current --squash-hierarchy --dest-dir ./docs/plugins $(NAMESPACE).$(NAME)
 	make -C docs html
 
+vendor:
+	git clone --depth=1 https://github.com/Apipie/apypie/ build/apypie-git
```

Comment:
Do we want to lock this to a version that gets set or updated ? Or just always take master?

Reply:
I would lock versions, just *right now* I need master for it to not explode ;)